### ISSUE TITLE
Download button redirecting to localhost

### ISSUE THREAD

**Issue by Kimbaras** [JMBeresford/retrom#141]
Hey!

First of all I confirm the previous bug (#127) is resolved, thanks!

Now a new one.
I've updated the docker image (v0.1.11 at the time of writing this) configured also the IGDB import (which earlier was still missing) for the most part seems fine, save for some games not recognized / identified by IGDB but this is not a retrom problem (but I'll talk about this later...).

I've noticed that clicking the "Download" button redirects the browser to a localhost address, which (most of the times) will fail (at least, for how I'm deploying such applications).

Clicking on this
![download_button](https://github.com/user-attachments/assets/71a1e795-c60e-456f-8fb5-9a5704036d0c)

Leads to this
![error](https://github.com/user-attachments/assets/5854119d-9b22-4004-87b2-3f5e9b11a25a)

I'm not really sure what's the desidered effect here, but I believe it could be something like "download the whole folder"? (I believe I've seen something like this applied in gameyfin but could be wrong)

So, here the issue is that it's redirecting to localhost and so it's not working.

Then there is the question: is my above assumption correct?
If not and if it's downloading just a specific file, how it's behaving while there are multiple files in the folder? (say the ISO as well as a readme or similar)

Thanks!

**Comment by JMBeresford** [comment]
Ah yes, I know what the issue is here. Will resolve shortly thanks for bringing it up!

To answer your question, the intended behavior is to download the each of the game's files, zipped. This behavior is only for the web client, the desktop client will 'install' the game files to a directory managed by Retrom, exactly as they are stored in the server.

**Comment by JMBeresford** [comment]
@Kimbaras Currently there is no third-party installer/setup launching, or arbitrary executable launching supported. You need a valid emulator configuration + profile ( as per the [emulator config docs](https://github.com/JMBeresford/retrom/blob/main/docs/emulator-config/README.md) ) to be able to launch installed games in the desktop client.

tldr; only emulated games are currently supported, and require emulator configs to be available

Native (PC, macOS, Linux) game support is incoming in the very near future, follow #106 for progress on that
